README.md

```diff
@@ -1,10 +1,12 @@
-# CompoundKit API
+# Compound Kit API
 
 [![Lint](https://github.com/dinngo/compound-kit-api/actions/workflows/lint.yml/badge.svg)](https://github.com/dinngo/compound-kit-api/actions/workflows/lint.yml)
 [![Unit Test](https://github.com/dinngo/compound-kit-api/actions/workflows/unit-test.yml/badge.svg)](https://github.com/dinngo/compound-kit-api/actions/workflows/unit-test.yml)
 [![E2E Test](https://github.com/dinngo/compound-kit-api/actions/workflows/e2e-test.yml/badge.svg)](https://github.com/dinngo/compound-kit-api/actions/workflows/e2e-test.yml)
 
-The API service for CompoundKit
+Compound Kit empowers developers to rapidly build intent-centric applications and enhances the user experience for the Compound protocol. This repository includes the Compound Kit API service.
```

Comment:
Could mention that suggest developers considering using https://github.com/dinngo/compound-kit-js-sdk.